src/spring/azext_spring/_params.py

```diff
@@ -402,6 +403,10 @@ def prepare_logs_argument(c):
             c.argument('backend_protocol',
                        arg_type=get_enum_type(BackendProtocol),
                        help='Ingress backend protocol of app.')
+            c.argument('client_auth_certs',
+                       type=str,
+                       validator=validate_ingress_client_auth_certificates,
+                       help="A space-separated string containing resource ids of certificates for client authentication. e.g: --client_auth_certs='id0 id1'. Use '' to clear existing certificates.")
```

Comment:
`type=str` is the default setting, so it does not need to be explicitly specified
```suggestion
            c.argument('client_auth_certs',
                       validator=validate_ingress_client_auth_certificates,
                       help="A space-separated string containing resource ids of certificates for client authentication. e.g: --client_auth_certs='id0 id1'. Use '' to clear existing certificates.")
```

Reply:
thanks, updated